odoo/addons/base/models/res_config.py

```diff
@@ -797,3 +799,9 @@ def create(self, vals_list):
                     vals.pop(field.name)
 
         return super().create(vals_list)
+
+    def action_open_template_user(self):
+        action = self.env["ir.actions.actions"]._for_xml_id("base.action_res_users")
+        action['res_id'] = literal_eval(self.env['ir.config_parameter'].sudo().get_param('base.template_portal_user_id', 'False'))
```

Comment:
AFAIK, it's supposed to be an int, is literal_eval really necessary here ? :thinking:

Reply:
And do we intend to land on a void form view if the config parameter is not set ? Shouldn't we check it exists (as it is not synchronized, user could have been removed) ?